homeassistant/components/modbus/sensor.py

```diff
@@ -63,6 +63,9 @@ def number(value: Any) -> Union[int, float]:
         raise vol.Invalid(f"invalid number {value}") from err
 
 
+""" Add bit option for single bit extraction from registry """
+CONF_BIT = 'bit'
```

Comment:
constants belong in const.py

Reply:
Thanks for tips, I moved on const.py file and upload the new code